packages/react-query/src/__tests__/useQuery.promise.test.tsx

```diff
@@ -59,6 +59,11 @@ describe('useQuery().promise', () => {
         },
       })
 
+      // Force Page to re-render when query state changes
+      React.useEffect(() => {
+        // noop
+      }, [query.status])
+
```

Comment:
why is this necessary ?

Reply:
The below test with [Page, MyComponent] only had one element returned otherwise

Reply:
would ist work if we’d just render the status in a div instead:

```
<div>status: {query.status}</div>
```

Reply:
this is in a `describe('useQuery().promise', () => {` suite, so it probably becomes a redundant test if we don't use query.promise.

Reply:
yeah I meant additionally, just instead of the effect

Reply:
That works.